lib/pages/chat/chat.dart

```diff
@@ -1287,3 +1293,8 @@ class ChatController extends State<Chat>
 }
 
 enum EmojiPickerType { reaction, keyboard }
+
+class FowardAgr {
```

Comment:
What is this ??

Reply:
we have `copilot` and `autocomplete` please use the full name :))

Reply:
I'll remove it